packages/web/src/pages/sign-up-page/components/layout.tsx

```diff
@@ -227,7 +227,7 @@ export const PageFooter = (props: PageFooterProps) => {
         fullWidth
         isLoading={isSubmitting}
         css={!isMobile && centered && { width: 343 }}
-        disabled={!dirty || !isValid}
```

Comment:
I swapped to `touched` instead of `dirty` since after adding in the logic to where you can close and re-enter your previous stuff it made sense to allow submits if their values are still good (most noticeably on the handle field)